tests/test_hooks.py

```diff
@@ -486,6 +486,9 @@ def test_terminal_summary_server_success(self, isolate_ibutsu_env_vars):
         mock_plugin.enabled = True
         mock_plugin.is_s3_mode = False
         mock_plugin.is_server_mode = True
+        mock_plugin.ibutsu_project = "test-project"
+        # Mock project_uuid property to return a resolved UUID
+        mock_plugin.project_uuid = "12345678-1234-1234-1234-123456789abc"
```

Comment:
Great update! Consider extending the test coverage a bit more here.

Mock the `project_api.get_project_list(filter=[f"name={project_value}"])` call and let the plugin instance's property actually resolve a value instead of mocking the value itself.